python/tvm/relax/op/mask.py

```diff
@@ -34,5 +34,5 @@ def masked_fill(x: Expr, mask: Expr, value: Expr):
     result : relax.Expr
         The filled tensor.
     """
-    values = _ffi_api.full_like(x, value)  # type: ignore
+    values = _ffi_api.full_like(x, value, x.struct_info.dtype)  # type: ignore
```

Comment:
Here it’s better to use the Python interface of `full_like` than the FFI interface, because the Python interface has clear semantics on the dtype, and the dtype can be None, which means being the same as the input dtype.

https://github.com/mlc-ai/relax/blob/8ee979dd8543f1de523458243f9ccbca38cacc31/python/tvm/relax/op/create.py#L56-L79

So here we can do `from .create import full_like`, and use `values = full_like(x, value)`

Reply:
Thanks for your suggestion, please check the update.